pkg/test/pods.go

```diff
@@ -137,13 +144,19 @@ func Pod(overrides ...PodOptions) *v1.Pod {
 			Phase:      options.Phase,
 		},
 	}
-	if options.InitImage != "" {
-		p.Spec.InitContainers = []v1.Container{{
-			Name:      RandomName(),
-			Image:     options.InitImage,
-			Resources: options.InitResourceRequirements,
-		}}
+	if options.Overhead != nil {
+		p.Spec.Overhead = options.Overhead
 	}
+	if options.InitContainers != nil {
+
+		for _, init := range options.InitContainers {
+			init.Name = RandomName()
+			init.Image = InitImage
```

Comment:
You can probably just keep this as 
```suggestion
			init.Image = "pause"
```